Sources/XcbeautifyLib/Renderers/TeamCityRenderer.swift

```diff
@@ -0,0 +1,262 @@
+import Colorizer
+import Foundation
+
+struct TeamCityRenderer: OutputRendering {
```

Comment:
As a note, there are many default implementations in `OutputRendering`. Have you considered if any of those need to be overridden for this renderer?

Reply:
Yes, I implemented all the functions that were needed, and the rest of the implementation should have remained as before in TerminalRenderer